public/cli/lib/rendering/renderFrames.ts

```diff
@@ -33,6 +33,11 @@ export async function getHtml(
     animationSettings.background.file && animationSettings.background.type == 'image'
       ? await DataURI(animationSettings.background.file)
       : null;
-  const data = { timings, fps, animationSettings, backgroundDataUri };
+  let data = {
+    timings: JSON.stringify(timings),
+    fps: fps,
+    animationSettings: animationSettings,
+    backgroundDataUri: backgroundDataUri,
```

Comment:
In TS, this can be simplified.  If the names of key and value are the same, TS allows a shortcut notation of only specifying the key. e.g. `fps, animationSettings, backgroundDataUri };`.